src/components/AnswersOptions/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React, {useState} from 'react';
+import {View} from 'react-native';
+import RadioForm, {
+  RadioButtonInput,
+  RadioButtonLabel,
+} from 'react-native-simple-radio-button';
+import {AnswerRadioContainer, styles, ViewMarginRight} from './styles';
+import {Color} from 'theme/colors';
+
+type AnswersOptionsPropsType = {
+  onPress: (value: number) => void;
+  data: string[];
+};
+
+export const AnswersOptions = ({onPress, data}: AnswersOptionsPropsType) => {
+  const transformData = data.map((el, i) => ({label: el, value: i}));
```

Comment:
why we need this?

Reply:
Because the data format answers will change, and only the array will come to us